banzai/main.py

```diff
@@ -288,9 +288,9 @@ def update_db():
         logger.error('Could not populate instruments table: {error}'.format(error=logs.format_exception()))
 
 
-def add_bpm():
-    parser = argparse.ArgumentParser(description="Add a bad pixel mask to the db.")
-    parser.add_argument('filepath', help='Full path to Bad Pixel Mask file')
+def add_cal():
+    parser = argparse.ArgumentParser(description="Add a calibration file (bad pixel mask, bias, dark, skyflat) to the db.")
```

Comment:
"Add a super calibration file to the db." maybe?

I generally try to avoid hard-coding lists of of the type of calibrations since they can get stale if we were to add a new one and not update it.